app/Services/Manager.php

```diff
@@ -189,19 +204,31 @@ public function collectContestInfo($data)
      *
      * @return object $signups
      */
-    public function getUserSignup($id, $campaign = null, $campaignRun = null)
+    public function getUserSignup($id, $campaign = null, $campaignRun = null, $getMultiple = false)
     {
         $signups = $this->northstar->getUserSignups($id, $campaign);
 
+        if ($getMultiple) {
+            $multipleSignups = [];
+        }
```

Comment:
no, the problem is originally this function would loop through all of the results, but it would only record them into one variable. thus, if i wanted return multiple signups, i cant.

$multipleSignups indicates we should return an array of results, instead of just one